go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/gorilla/websocket
 
-go 1.12
+go 1.19
```

Comment:
With the jump to 1.19, there's no need for the [< 1.16 version of the doHandshake function](https://github.com/gorilla/websocket/blob/main/tls_handshake_116.go).  Consider [inlining](https://github.com/gorilla/websocket/blob/80393295c1185e50d0b784d4bc5ffaa918d187b9/client.go#L350)  the [>= 1.17 version of the function](https://github.com/gorilla/websocket/blob/main/tls_handshake.go) and deleting files tls_handshake*.go.